Makefile

```diff
@@ -0,0 +1,41 @@
+.PHONY: help build fmt lint vet run test style cyclo
+
+SOURCES:=$(shell find . -name '*.go')
+
+default: build
+
+build: ## Run go build
+	@go build
+
+fmt: ## Run go fmt -w for all sources
+	@gofmt -w $(SOURCES)
+
+lint: ## Run golint
+	@go get golang.org/x/lint/golint
+	@if	golint ./... | grep -v ALL_CAPS | grep .; then \
```

Comment:
This is weird but it reflects original `.travis.gofmt.sh` file. I don't know why `golint ./... | grep -v <whatever>` returns exit code 1 instead of 0.